<?xml version="1.0"?>

<launch>
    
    <node pkg="points_process" type="points_process_node" name="points_process_node" output="screen">
        
        <param name="sub_topic" value="/pandar_points"/>
        <param name="pub_topic" value="/pandar_points_processed"/>
        
        <param name="crop_view_mode" value="false"/>
        <param name="crop_range_mode" value="true"/>
        <param name="filter_mode" value="false"/>
        <param name="downsample_mode" value="true"/>
        <param name="show_points_size" value="true"/>
        <param name="show_time" value="true"/>
        
        <param name="view_number" value="1"/>
        <param name="field_of_view" value="100"/>

        <param name="sensor_height" value="2.0"/>
        <param name="view_higher_limit" value="4.0"/>
        <param name="view_lower_limit" value="-4.0"/>
        <param name="min_distance" value="1.5"/>
        <param name="max_distance" value="50.0"/>
        
        <param name="meank" value="5"/>
        <param name="stdmul" value="0.2"/>
        
        <param name="leafsize" value="0.15"/>
        
    </node>

</launch>
